assets/js/components/Dashboard/PendingScreenDetail.tsx

```diff
@@ -0,0 +1,32 @@
+// Note: component name is a bit unclear--this can show detail for both pending and live screens,
+// but its presentation is particular to the pending screens page.
+
+import React from "react";
+import { Screen } from "../../models/screen";
+import { ScreenConfiguration } from "../../models/screen_configuration";
+import ScreenSimulation from "./ScreenSimulation";
+import { fetchExistingScreens } from "../../utils/api";
+
+interface Props {
+  config: ScreenConfiguration;
+  // I'm still not super clear what the purpose of this is, or how/where to store it...
+  // We now have two different ways of hiding screens from the places list:
+  // - When hidden via the old admin tool, the screen is excluded from places_and_screens.json
+  // - When hidden this new way, the screen is known to Screenplay--and thus needs a full, valid
+  //   config + Screenplay-relevant metadata, since we show info about it here on
+  //   the pending page--but ignored when rendering the places list.
+  //
+  // This seems messy and I'm not sure how to explain the issue to design.
+  isHiddenOnPlacesPage: boolean;
```

Comment:
I'm not sure this is necessary. By default, screens in the pending config will be hidden from the places page. Maybe you can just look up the Screenplay-relevant metadata and pending screens separately instead of needing to keep pending screens in the place object? Looking at the designs, the only metadata you need from `places_and_screens` is the ID and name. The rest could be ignored.

Reply:
Actually, looks like I misread the design. There are live screens shown on this page. But that makes me wonder, should there be? It feels like only pending screens should live here so that way the user doesn't have to wonder what kind of screen they are looking at if they happen to overlook the status.